projects/smoke_ads1259/src/main.c

```diff
@@ -1,15 +1,23 @@
+#include <stdlib.h>
+
 #include "ads1259_adc.h"
```

Comment:
Should have `#include "adc1259_adc.h"` at the top, but not a big deal.

Reply:
Not necessarily since that's not the respective header file for the module, right? just following clang format

Reply:
Oh yeah 🤦 , thought this was a test for some reason.